src/order/upgrade/privateCloud/order-upgrade-privateCloud.v6.service.js

```diff
@@ -0,0 +1,37 @@
+angular
+    .module("ovh-api-services")
+    .service("OvhApiOrderUpgradePrivateCloudV6", function ($resource, $cacheFactory) {
+
+        "use strict";
+
+        // Cache to invalidate
+        var queryCache = $cacheFactory("OvhApiOrderUpgradePrivateCloudV6Query");
+        var cache = $cacheFactory("OvhApiOrderUpgradePrivateCloudV6");
+
+        var interceptor = {
+            response: function (response) {
+                resource.resetQueryCache();
+                return response.data;
+            }
+        };
+
+        var resource = $resource("/order/upgrade/privateCloud/:serviceName/:planCode", {
+            serviceName: "@serviceName",
+            planCode: "@planCode"
+        }, {
+            getServices: { method: "GET", cache: queryCache, isArray: true, url: "/order/upgrade/privateCloud" },
+            getOffers: { method: "GET", cache: queryCache, isArray: true, url: "/order/upgrade/privateCloud/:serviceName" },
```

Comment:
Imo, this two should be in a `OvhApiOrderUpgradePrivateCloud` with `getServices` being a `query` and `getOffers` a `get` as it is the way it is done elsewhere (it is easier to know which call to make given the API path) but maybe it doesn't make sense to do it that way anymore/ the naming is not explicit enough